ros/lib/utils.py

```diff
@@ -135,10 +134,11 @@ def sort_io_dict(performance_utilization: dict):
 
 
 def system_ids_by_org_id(org_id, fetch_records=False):
-    account_query = db.session.query(RhAccount.id).filter(RhAccount.org_id == org_id).subquery()
+    account_query = select(RhAccount.id).where(RhAccount.org_id == org_id)
+
     if fetch_records is True:
-        return db.session.query(System).filter(System.tenant_id.in_(account_query))
-    return db.session.query(System.id).filter(System.tenant_id.in_(account_query))
+        return select(System).filter(System.tenant_id.in_(account_query))
+    return select(System.id).filter(System.tenant_id.in_(account_query))
```

Comment:
I didn't really get why we have replaced `db.session.query` with `select` in some places but not all.

Reply:
If you see the code flow then `system_ids_by_org_id` is being used as input for other method as example https://github.com/RedHatInsights/ros-backend/blob/main/ros/lib/utils.py#L179 and if we revert this there sqlalchemy starts throwing warning about this type of passing db.session.query object wont be supported in future